package.json

```diff
@@ -13,7 +13,7 @@
   ],
   "scripts": {
     "build": "cross-env NODE_ENV=prod webpack",
-    "start": "cross-env NODE_ENV=dev webpack-dev-server --progress --debug --config --display-error-details",
+    "start": "cross-env NODE_ENV=dev webpack-dev-server --progress --debug --config ./webpack.config.js",
```

Comment:
Thanks for the feedback. I didn't know about this. I'll go ahead and update this this week.